tensorflow_model_optimization/python/core/clustering/keras/cluster.py

```diff
@@ -14,6 +14,9 @@
 # ==============================================================================
 """Clustering API functions for Keras models."""
 
+import distutils.version
```

Comment:
this is leftover after resolving conflicts, i don't think we use it here.
Could we please remove it ? thanks